src/components/LoanCards/Buttons/LendButton.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<kv-button
-		v-kv-track-event="['Lending', 'Add to basket', 'lend-button-click', loanId, loanId]"
+		v-kv-track-event="['Lending', eventInfo, 'lend-button-click', loanId, loanId]"
```

Comment:
I don't think we should be changing this event. As mentioned in the slack thread we should have a new and separate even in the LendAmountButton component.

Changes to this event will cascade through an untold number of automated metrics dashboards that look at overall add to basket activity.